src/Shared/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -164,6 +169,7 @@ internal void CopyPropertiesTo(ApplicationInsightsServiceOptions target)
                 target.InstrumentationKey = this.InstrumentationKey;
             }
 
+            target.ConnectionString = this.ConnectionString;
```

Comment:
do you need to check constring for null or empty?

Reply:
this isn't necessary. it won't affect the behavior to copy the properties as is from one ServiceOptions to the other